protocol_http.go

```diff
@@ -33,6 +33,10 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
+const (
+	contentApplicationPrefix = "application/"
+)
```

Comment:
This belongs with the other constants over in `protocol_connect.go`: it's not used at all in this file but is used numerous times there.

Also, perhaps name it `contentConnectUnaryPrefix` and change the other one to `contentConnectStreamPrefix`?

Reply:
It's used in the rest protocol which is what caused the split. Didn't want the constant to have connect annotation when its a HTTP thing. wdyt?

Reply:
Separated out to separate const for each.